.github/workflows/release-embedding-sdk.yml

```diff
@@ -102,6 +114,12 @@ jobs:
           name: metabase-sdk
           path: ./resources/embedding-sdk
 
+      - name: Upload changelog
```

Comment:
We'll use this when checking out `master` later when we're about to create a PR that update `master` with the latest changelog.